README.md

```diff
@@ -21,3 +21,18 @@ and adapted in `users.json`.
 from your gitlab instance.
 If you want to build your image locally (for example because you want to change the pre-configured
 users in users.json), run `docker build . -t renku/renku-demo`
+
+### Running demo against any renku instance
+Exmaples:
+- `docker run -it --rm --network host \`<br> 
+   `-e GITLAB_URL=http://$(minikube ip)/gitlab \`<br> 
+   `-e KEYCLOAK_URL=http://$(minikube ip)  \`<br> 
+   `renku/renku-demo:latest`
+
+- `docker run -it --rm --network host \`<br> 
+   `-e GITLAB_URL=https://staging.testing.datascience.ch/gitlab \`<br> 
```

Comment:
maybe don't show the staging URL in public?

Reply:
Ok, didn't see this as a problem but I will remove it.